Translator/frmTranslate.cs

```diff
@@ -287,22 +273,6 @@ private void dgvSection_CellValueChanged(object sender, DataGridViewCellEventArg
                             element.InnerText = strNameOnPage;
                         }
                     }
-
-
-                    XmlAttribute objAttrib = xmlNodeLocal.Attributes?["translated"];
-                    if (objAttrib != null)
-                    {
-                        if (!flag)
-                            xmlNodeLocal.Attributes.Remove(objAttrib);
-                        else
-                            objAttrib.InnerText = bool.TrueString;
-                    }
-                    else if (flag)
-                    {
-                        objAttrib = _objDataDoc.CreateAttribute("translated");
-                        objAttrib.InnerText = bool.TrueString;
-                        xmlNodeLocal.Attributes?.Append(objAttrib);
-                    }
```

Comment:
This needs to be rethought. The `translated` tag is there for if something's English name matches its translated name, which is the case sometimes in German and French. Since stuff without a translation gets highlighted in a different color, this makes sure those highlights don't happen when they aren't supposed to.

Reply:
Another fix for the same reason.